build.sh

```diff
@@ -197,7 +197,12 @@ test_ios_static() {
 
     destination="$1"
     xc "-scheme 'Realm iOS static' -configuration $CONFIGURATION -sdk iphonesimulator -destination '$destination' build"
-    xc "-scheme 'Realm iOS static' -configuration $CONFIGURATION -sdk iphonesimulator -destination '$destination' test 'ARCHS=\$(ARCHS_STANDARD_32_BIT)'"
+    if [[ `echo $REALM_XCODE_VERSION | awk '{tmp=match($0, /^[[:digit:]]+/); print substr($0, RSTART, RLENGTH)}'` -lt 9 ]]; then
+        xc "-scheme 'Realm iOS static' -configuration $CONFIGURATION -sdk iphonesimulator -destination '$destination' test 'ARCHS=\$(ARCHS_STANDARD_32_BIT)'"
+    else
+        # in Xcode 9 iOS 11 is the standard, and it does not support 32-bit iOS configurations
+        xc "-scheme 'Realm iOS static' -configuration $CONFIGURATION -sdk iphonesimulator -destination '$destination' test"
```

Comment:
We test the 64-bit configuration below (~line 213). This looks like it'll result in it being tested twice for Xcode 9.